src/azure-cli/azure/cli/command_modules/network/commands.py

```diff
@@ -808,6 +808,10 @@ def _make_singular(value):
         g.show_command('show')
         g.command('list', 'list')
 
+    with self.command_group('network express-route peering connection config', network_erconn_sdk) as g:
+        g.custom_command('add', 'add_express_route_peering_connection_config')
```

Comment:
Please use `set` command instead of `add`. Because it's not a collection property.